### ISSUE TITLE
Creating an App Icon for the Electron Desktop App

### ISSUE THREAD

**Issue by natalialuzuriaga** [DSACMS/dedupliFHIR#61]
**Is your feature request related to a problem? Please describe.**
As part of shipping the Electron desktop app, an app icon is needed.

**Describe the intended behavior**
The app icon design should represent the purpose and functionality of DedupliFHIR

**Describe the solution you'd like**
Design an app icon for the DedupliFHIR app using Figma and other design tools!

**Comment by natalialuzuriaga** [comment]
Comments from Slack for ref: 

I have no ounce of artistic skill in me so I'm glad you are picking up this ticket 🙏 
Throwing out some random ideas: DedupliFHIR is all about identifying duplicates in patient records so some sort of visual that depicts that would work. Since our target users are Accountable Care Organizations (ACOs), maybe incorporating something with ACOs into it would be cool too. Not sure if we explained this well in our meetings but the "[FHIR](https://www.altexsoft.com/blog/fhir-standard/)" part in the tool's name has to do with the data format of the patient records, so that could be included too. While working on this project, I've seen way too many fhir puns using FHIR 🔥 

I was leaning more towards a flat 2D vector style but honestly that's a personal preference. If you're feeling 3D then go for it.

If you want to play around with the tool, lmk! We can set up a meeting to get that installed in your local machine and I can give you more of a deep dive into it too

From Electron docs: It's recommended to start with a 1024x1024px image before converting it to various sizes

**Comment by decause-gov** [comment]
06/17:
- Working in Figma
- IDEA: Create a OSPO folder, perhaps a DedupliFHIR folder within that
- See above for Nat's design ideas

**Comment by CreativeNick** [comment]
### Useful Links
https://developer.apple.com/design/human-interface-guidelines/app-icons
https://developer.apple.com/design/resources/
https://www.figma.com/community/file/1040708197994685442

### Link to Figma
https://www.figma.com/design/OR5A3ttvvvTG9U3bOEicvB/DedupliFHIR-Mockups?node-id=179-80&t=1jbtXrZb3aUNMbnF-1
(Note: may not be able to view due to sharing access, check with @natalialuzuriaga if you need access)

**Comment by decause-gov** [comment]
@CreativeNick Update this ticket upon your return plz :)

**Comment by natalialuzuriaga** [comment]
06/24:
- Ideas for more mockups based on feedback from team, iterations for the non-transparent backgrounds
- Will attempt to add the blue to records

**Comment by CreativeNick** [comment]
06/27:
- Made new variations + Remy's idea, waiting on additional feedback/comments @natalialuzuriaga @IsaacMilarky @decause-gov

**Comment by decause-gov** [comment]
<img width="151" alt="Screenshot 2024-06-28 at 10 42 38 AM" src="https://github.com/DSACMS/dedupliFHIR/assets/107957201/df8a628d-dfcd-4b5e-b416-b4f5e6f93ecf">

🔥 🔥

**Comment by IsaacMilarky** [comment]
6/28:

- Made new design seen above Kudos to Remy
- ACTION: Get broader feedback from Aayat

**Comment by IsaacMilarky** [comment]
7/1:
- ACTION: Follow up with @aayatsali on slack asap
- Hopefully will be approved and be added to repo soon

**Comment by CreativeNick** [comment]
7/1:
- Icon decided based on vote
- Will begin implementing icon in [new ticket](https://github.com/DSACMS/dedupliFHIR/issues/68)

Icon chosen (this is the MacOS version):
![Image](https://github.com/DSACMS/dedupliFHIR/assets/43157506/c461faec-5830-43cd-8835-037fd4e73cf7)
